src/core/MenuSelect/index.stories.tsx

```diff
@@ -0,0 +1,302 @@
+import ButtonBase from "@material-ui/core/ButtonBase";
+import Popper from "@material-ui/core/Popper";
+import { makeStyles } from "@material-ui/core/styles";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import { AutocompleteCloseReason } from "@material-ui/lab";
+import { Args, Story } from "@storybook/react";
+import React from "react";
+import Chip from "../Chip";
+import MenuSelect, { DefaultMenuSelectOption } from "./index";
+
+const Demo = (props: Args): JSX.Element => {
+  // eslint-disable-next-line react/prop-types
+  const { multiple, options = GITHUB_LABELS, search } = props;
+
+  const classes = useStyles();
+
+  const [anchorEl, setAnchorEl] = React.useState<HTMLElement | null>(null);
+
+  const [value, setValue] = React.useState<
```

Comment:
why not import useState from react? (no need for a change, curious on your reasoning)

Reply:
Oh you're right lol Totally missed that 😆 Thanks for pointing out!

I've been using https://copilot.github.com/ in VS Code and it autocompleted all of my useState declarations with `React.useState` for no good reason lol